### PR TITLE
[#147] rename tagged to tags

### LINKED ISSUE #147: Coding standard violation on Storage component
In the file src/main/java/seedu/address/storage/JsonAdaptedPerson.java on the master directory, I found out that the variable `tagged` is not in plural even though it represents a collection of objects, hence violating CS2103T coding standard.

Below shows lines 33-47 of the code for easy reference.

```java
    /**
     * Constructs a {@code JsonAdaptedPerson} with the given person details.
     */
    @JsonCreator
    public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
            @JsonProperty("email") String email, @JsonProperty("address") String address,
            @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
        this.name = name;
        this.phone = phone;
        this.email = email;
        this.address = address;
        if (tagged != null) {
            this.tagged.addAll(tagged);
        }
    }
```

Here is the relevant reference in the SE-EDU Java coding standard.
image

### PR DESCRIPTION
**Pull request by Eclipse-Dominator**
Fixes #147 

Change the variable tagged to tags in the affected class [took over from #153]
Change the json property tagged to tags in all affected save files
(src/main/java/seedu/address/storage/JsonAdaptedPerson.java).

image

### CODE REVIEW

File: src/main/java/seedu/address/storage/JsonAdaptedPerson.java [hunk 1]
```diff
@@ -54,19 +54,21 @@ public JsonAdaptedPerson(Person source) {
...
-        tagged.addAll(source.getTags().stream()
+        tags.addAll(source.getTags().stream()
                 .map(JsonAdaptedTag::new)
                 .collect(Collectors.toList()));
     }
 
     /**
-     * Converts this Jackson-friendly adapted person object into the model's {@code Person} object.
+     * Converts this Jackson-friendly adapted person object into the model's
+     * {@code Person} object.
```
Comment (damithc) [inline src/main/java/seedu/address/storage/JsonAdaptedPerson.java]: Unrelated change?

File: src/main/java/seedu/address/storage/JsonAdaptedPerson.java [hunk 2]
```diff
@@ -54,19 +54,21 @@ public JsonAdaptedPerson(Person source) {
...
      *
-     * @throws IllegalValueException if there were any data constraints violated in the adapted person.
+     * @throws IllegalValueException if there were any data constraints violated in
+     *                               the adapted person.
```
Comment (damithc) [inline src/main/java/seedu/address/storage/JsonAdaptedPerson.java]: Unrelated changed?

**Review by damithc — CHANGES_REQUESTED** [review]
1. Some minor nits suggested.
1. The new commit is missing a full commit message, compliant with https://se-education.org/guides/conventions/git.html#commit-message-body
1. Did you do some manual tests to verify the tagging still works?

File: src/main/java/seedu/address/storage/JsonAdaptedPerson.java (hunk 1 above)
Comment (Eclipse-Dominator) [reply src/main/java/seedu/address/storage/JsonAdaptedPerson.java]: I have updated my formatter to only format source changes.

**Comment by Eclipse-Dominator** [comment]
I have updated the commit to include a commit body. I did do some testing with changes and no issues were spotted.

**Review by damithc — CHANGES_REQUESTED** [review]
Code looks fine now. The commit messages can be improved further.

```
JsonAdaptedPerson class: Change tagged to tags
........................
JsonAdaptedPerson class has a list object that is named tagged.

A list object represents a collection of objects hence should be in
plural form based on the coding standard.

Let's change the name of the list object from tagged to tags.
```

```
JsonAdaptedPerson: Use tags for JSON property (#169)
........................
Currently, "tagged" is used as the default JSON attribute when saving.

It is inconsistent with the new naming scheme.

Let's change the JSON attribute to be consistent with
the new naming scheme.
```

* The two commit subjects should be phrased similarly, as they are doing similar things.
* `Currently` can be omitted, as it is implied.
* These phrases raise questions `default JSON attribute` (_default_?), `new naming scheme` (what _new_ naming scheme?)

How  about the following?
```
JsonAdaptedPerson: Change variable name tagged -> tags
...............
The JsonAdaptedPerson class contains a list object called 'tagged'.

According to our coding standard, a variable holding a collection
of items should have a plural form name. 

Let's change the variable name to 'tags' instead.
```

```
JsonAdaptedPerson: Change JSON property name tagged -> tags
......................
Similar to the previous commit, let's change the JSON property name 
'tagged' in the JsonAdaptedPerson class to 'tags'.
```

**Review by damithc — CHANGES_REQUESTED** [review]
Almost there. You missed updating the subject of the commit 1 subject. :-p

**Comment by Eclipse-Dominator** [comment]
Yup I just realized it as well.

**Review by damithc — APPROVED** [review]
LGTM 👍

**Comment by damithc** [comment]
Thanks to @Py0000 for your contribution to this PR too.
